.github/reusable-build/action.yml

```
@@ -0,0 +1,38 @@
+name: Resusable steps to build tfx-bsl
+
+inputs:
+  python-version:
+    description: 'Python version'
+    required: true
+runs:
+  using: 'composite'
+  steps:
+
+  - name: Set up Python ${{ inputs.python-version }}
+    uses: actions/setup-python@v5
+    with:
+      python-version: ${{ inputs.python-version }}
+
+  - name: Upgrade pip
+    shell: bash
+    run: |
+      python -m pip install --upgrade pip pytest
+
+  - name: Build the package for Python ${{ inputs.python-version }}
+    shell: bash
+    run: |
+      version="${{ inputs.python-version }}"
+      docker compose run -e PYTHON_VERSION=$(echo "$version" | sed 's/\.//') manylinux2010
+
+  - name: Upload wheel artifact for Python ${{ inputs.python-version }}
+    uses: actions/upload-artifact@v4.4.0
+    with:
+      name: tfx-bsl-wheel-py${{ inputs.python-version }}
+      path: dist/*.whl
```

```suggestion
      path: dist/*.whl
      if-no-files-found: error
```

Making this fail explicitly will avoid errors down the line.